pkg/build/inline.go

```diff
@@ -64,19 +65,29 @@ func NewInlinerWithScheme(scheme files.URLScheme, objReader files.FileObjReader)
 	}
 }
 
-// BundleFiles converts dereferences file-references in for bundle files.
+// BundleFiles converts dereferences file-references in for bundle files. If
+// the bundlePath is defined and not absolute and the scheme is file based
+// scheme, then the path is made absolute before proceeding.
 func (n *Inliner) BundleFiles(ctx context.Context, data *bundle.BundleBuilder, bundlePath string) (*bundle.Bundle, error) {
 	bundleURL, err := url.Parse(bundlePath)
 	if err != nil {
 		return nil, err
 	}
+	bundleURL, err = n.PathRewriter.Abs(bundleURL)
```

Comment:
why is path rewriter a struct member as opposed to just a plain old function?

Reply:
Made a POF